cfn-resources/util/util.go

```diff
@@ -47,9 +48,29 @@ func CreateMongoDBClient(publicKey, privateKey string) (*mongodbatlas.Client, er
 	// Initialize the MongoDB Atlas API Client.
 	atlas := mongodbatlas.NewClient(client)
 	atlas.UserAgent = "mongodbatlas-cloudformation-resources/" + Version
+
+	basePath, err := newBasePath()
+
+	if err != nil {
+		return nil, err
+	}
+
+	if basePath != nil {
+		atlas.BaseURL = basePath
+	}
+
 	return atlas, nil
 }
 
+func newBasePath() (*url.URL, error) {
+	atlasURL := os.Getenv("MONGODB_ATLAS_OPS_MANAGER_URL")
+	if atlasURL == "" {
+		return nil, nil
+	}
+
+	return url.Parse(atlasURL)
+}
```

Comment:
This is the main change: We can now set the env variable `MONGODB_ATLAS_OPS_MANAGER_URL` (the same one that we use in terraform and AtlasCLI) to test the handler code against cloud-dev